memecity.engine.ui/UI/Overlay/OverlayTextItem.cpp

```diff
@@ -0,0 +1,9 @@
+﻿#include "OverlayTextItem.h"
+
+void OverlayTextItem::render()
```

Comment:
we hebben nu ook een `render_text` waarbij je direct de text en position kan meegeven